tests/ops/integration_tests/saas/test_simon_data_task.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from fides.api.models.policy import Policy
+from tests.ops.integration_tests.saas.connector_runner import ConnectorRunner
+
+
+@pytest.mark.integration_saas
+class TestSimonDataConnector:
+    def test_connection(self, simon_data_runner: ConnectorRunner):
+        simon_data_runner.test_connection()
+
+    async def test_access_request(
+        self, simon_data_runner: ConnectorRunner, policy, simon_data_identity_email: str
+    ):
+        access_results = await simon_data_runner.access_request(
+            access_policy=policy, identities={"email": simon_data_identity_email}
+        )
```

Comment:
Can you add an assertion to verify the email for the returned object matches the identity email?

Reply:
Adrian helped me with this! In this particular case we can remove this test as this connector is erasure only.